minet/cli/twitter/tweets.py

```diff
@@ -0,0 +1,77 @@
+# =============================================================================
+# Minet Twitter Tweets CLI Action
+# =============================================================================
+#
+# Logic of the `tw tweets` action.
+#
+import casanova
+from twitwi import (
+    normalize_tweet,
+    format_tweet_as_csv_row
+)
+from twitwi.constants import TWEET_FIELDS
+from twitter import TwitterHTTPError
+
+from ebbe import as_chunks
+
+from minet.cli.utils import LoadingBar
+from minet.twitter import TwitterAPIClient
+
+
+def twitter_tweets_hydration_action(cli_args):
```

Comment:
This action should, by convention, be called like the command so just `twitter_tweets_action`.